### PR TITLE
fix(toolkit): improve markdown for dark mode

### PR DESCRIPTION
**Pull request by knajjars**
Improves dark mode colors for markdown 
Also add `--turbo` flag for faster dev HMR

### CODE REVIEW

File: src/interfaces/assistants_web/src/styles/main.css [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-@tailwind utilities;
-
 @import url('./fonts.css');
 @import url('./globals.css');
 @import url('./code-style.css');
 @import url('./dark-mode.css');
 
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```
Comment (tomtobac) [inline src/interfaces/assistants_web/src/styles/main.css:8]: why this change? 🤔

File: src/interfaces/assistants_web/tailwind.config.js [hunk 2]
```diff
@@ -116,8 +116,6 @@ module.exports = {
         },
         DEFAULT: {
           css: {
-            'code::before': { content: '""' },
```
Comment (kravse) [inline src/interfaces/assistants_web/tailwind.config.js:119]: why remove this? This is old school stuff for `float`.

File: src/interfaces/assistants_web/src/styles/main.css (hunk 1 above)
Comment (knajjars) [reply src/interfaces/assistants_web/src/styles/main.css:8]: turbo pack requested this change

File: src/interfaces/assistants_web/tailwind.config.js (hunk 2 above)
Comment (knajjars) [reply src/interfaces/assistants_web/tailwind.config.js:119]: this removes the quotes that get added to `blockquotes` in markdown. We can keep them, just thought it would be cleaner if we displayed exactly what the model outputs

File: src/interfaces/assistants_web/src/styles/main.css (hunk 1 above)
Comment (tomtobac) [reply src/interfaces/assistants_web/src/styles/main.css:8]: if we make this change, the globals.css file won't work because its using the `theme`... wait a minute.. check `globals.css` we are re-importing again tailwind 🤦

File: src/interfaces/assistants_web/tailwind.config.js (hunk 2 above)
Comment (knajjars) [reply src/interfaces/assistants_web/tailwind.config.js:119]: oh wait, this I added on my previous PR. It was merged unintentionally so I am just reverting it

**Review by kravse — APPROVED** [review]

File: src/interfaces/assistants_web/tailwind.config.js (hunk 2 above)
Comment (kravse) [reply src/interfaces/assistants_web/tailwind.config.js:119]: 👍 i dunno what im talking about. haha.

File: src/interfaces/assistants_web/src/styles/main.css (hunk 1 above)
Comment (knajjars) [reply src/interfaces/assistants_web/src/styles/main.css:8]: oh nice cach
